common/src/helpers/formUtils.ts

```diff
@@ -57,3 +57,24 @@ export function enumToOptions<T>(
 export function getValues<T>(list: { value: T; label: string }[]): T[] {
   return list.map((x) => x.value);
 }
+
+/**
+ * Returns a list of strings (haystack) that match the word being searched for (needle).
+ * @param needle String that you want to search for.
+ * @param haystack List of strings to check against.
+ * @returns
+ */
+export function matchStringsCaseDiacriticInsensitive(
```

Comment:
Yea I added it in because I thought we would be using this to find the matching skills. However, I might of jumped the gun since we could be using a different method. I'll remove it till we figure it out.